shipctl/x86_64/WindowsServer_2016/utility.ps1

```diff
@@ -728,3 +786,106 @@ function _send_notification() {
     }
   }
 }
+
+function _send_irc_notification() {
+  param(
+    [string]$server,
+    [string]$channel,
+    [string]$nick,
+    [string]$password,
+    [string]$payload
+  )
+  $port = 6667
+
+  # make the connection
+  $client = New-Object -TypeName Net.Sockets.TcpClient
+  $client.Connect($server, $port)
+  $netStream = $client.GetStream()
+  $encoding = [System.Text.Encoding]::ASCII
+  $streamWriter = New-Object -Type System.IO.StreamWriter -ArgumentList $netStream, $encoding
+  $connection = "on"
+  # send initial login commands
+  $streamWriter.WriteLine("PASS $password")
```

Comment:
We should send this only if we have a value, right?

Reply:
it causes a "missing parameter" response from IRC and doesn't seem to impact the rest of the flow, but yeah i'll put a condition on there.